### ISSUE TITLE
Feature Request: Ability to add link to external data

### ISSUE THREAD

**Issue by msimanga** [schemaspy/schemaspy#181]
Firstly, thanks to all maintainers and contributors for your time and efforts.

Schemaspy does a good job of documenting the database. However, a lot of information relating to the database exist outside the database.  For various reasons, it is not always possible to add comments to the database. I am aware of the feature that allows you to use Markdown text in comments. 

In short, my request is for the ability to add a link to external documentation. I have two possible suggestiongs:

1. Add link to at the bottom _table_ documentation through an optional config property. It is up to me the user to ensure that my external documentation structure matches config option specified in my schemaspy .

 `schemaspy.external_doc_link="http://servername/{{schema}}/{{table_name}}"` 

2. Allow the user to maintain a separate list of tables and even fields in a **_separate config file_** in a specified format. This file can be passed as a paramater. Example if format is CSV would have contents below.

|schema|table|comment|
|-------|------|----------|
|dbo|my_table|Detailed documentation can be found [here]("http://servername/...)|
|dbo|table2|Detailed documentation can be found [here]("http://servername/...)|

The comment text would be appended to the documentation coming from the database.  I am a SQL developer with very little Java development. I hope this makes sense.

**Comment by npetzall** [comment]
There is schema metadata in xml which contains comments. We could check if that could be markdown then you could handle it thru a schemametadata xml-file. 

I haven't used it, but I could have a look at it.

**Comment by msimanga** [comment]
Thank you very much for pointing in the right direction. I tested using a meta file. I can confirm in my case the additional information was added to the output. Markdown included in file was parsed.  You need to use _**-meta**_ to point to your meta file. 

I used RC1. My sample file with columns renamed:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<schemaMeta xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:noNamespaceSchemaLocation="http://schemaspy.sourceforge.net/xmlschema/2011/02/05/schemaspy.meta.xsd">
   <comments>
      This is additional schema information.
   </comments>
	<tables>
      <table name="my_table_name" comments="This addtional table information and here is [link](http://example.com) to more information">
          <column name="field1" comments="Click this [link](http://example.com) for more information" />
          <column name="field2" comments="This comment for second column  Follow link for further documentation [link](http://example.com)" />
      </table>
   </tables>
</schemaMeta>
```

**Comment by npetzall** [comment]
So your good with that solution? Don't close we can use it as a documentation issue.

**Comment by msimanga** [comment]
Yes I am good with solution, it covers my feature request.

**Comment by rafalkasa** [comment]
@msimanga and @npetzall thank you for your time and verification. I would like to include this in SchemaSpy documentation because it can be used by others to store the database fields and tables comments outside the database.

Thank you for your contribution. Contribution is not only change the code :smile:

**Comment by npetzall** [comment]
@rafalkasa Do you have any idea about what's missing in the documentation?
Example with markdown and link?

**Comment by npetzall** [comment]
Closed by #349
